README.markdown

```diff
@@ -55,19 +55,19 @@ however, your first post might be moderated. This is simply to prevent spam.
 
 ### Getting Started
 
-1. Install the Glasgow Haskell Compiler (GHC) version 7.6 or higher.
+1. Install [Stack](http://docs.haskellstack.org/en/stable/README/)
 
 2. Change to the directory containing this document.
 
-3. Execute the command `ghci`, which will compile and load all the source code.
-   You may need to set permissions on the root directory and the ghci configuration
-   file, `chmod 600 .ghci ./`.
+3. Execute the command `stack setup`, which will install the appropriate compiler for this project.
 
-4. Inspect the introductory modules to get a feel for Haskell's syntax, then move
+4. Execute the command `stack exec ghci` which will compile and load all the source code
```

Comment:
running `stack ghci` gives:

```
<interactive>:1:6-66: error: Not in scope: ‘>>’

<interactive>:1:70-131: error: Not in scope: ‘>>’

<interactive>:1:135-196: error: Not in scope: ‘return’
```

Running `stack exec ghci` lands you in prelude, would love some pointers on how to solve this.